ServerCore/SyncController.cs

```diff
@@ -244,11 +234,15 @@ private string GetSiteVersion(Puzzle puzzle)
             foreach (var solvedPuzzle in solves) {
                 // If the request is asking whether certain puzzles are solved, check if it's
                 // in the set and, if so, put it in the list of solved puzzles to inform the
-                // requester of.
+                // requester of.  Also, if we've been asked to query all puzzles in the excluded
+                // group and this is one of them, put it in the list of solved puzzles.
```

Comment:
Do I want to know why? Seems counterintuitive but I don't know what the excluded group is.

Reply:
Good point.  I'll rename that from "excluded group" to something that conveys "group this puzzle is part of".  (It was "excluded" before because it doesn't count toward solve count.)